### PR TITLE
Upgrades gems with vulnerabilities, and their dependencies.

### PR DESCRIPTION
**Pull request by afred**

### CODE REVIEW

**Comment by afred** [comment]
@mccalluc sanity check and merge please?

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ source 'http://rubygems.org'
 gem 'rails', '~> 3.2.16'
 gem 'syck', platforms: [:ruby_20]
 
+# gem 'activeadmin'
 gem 'activeadmin'
```
Comment (mccalluc) [inline Gemfile]: @afred: do you mean to comment out activeadmin?

Comment (afred) [inline Gemfile]: @mccalluc i think one of those was pinned to a version or branch or something. But it's getting included on the following line.

File: Gemfile [hunk 2]
```diff
@@ -18,11 +19,12 @@ gem 'sass', '~> 3.2.13'
...
 gem 'compass'
 gem 'devise'
-gem 'jquery-rails'
+gem 'jquery-rails', '~> 3.1.3'
+gem 'jquery-ui-rails'
 gem 'kaminari'
 gem 'mysql2'
 gem 'progressbar'
-gem 'rsolr-footnotes'
+# gem 'rsolr-footnotes'
```
Comment (mccalluc) [inline Gemfile]: Were we just not using this?

**Pull request merged**